jdbc/src/test/java/org/apache/metamodel/jdbc/dialects/PostgresqlQueryRewriterTest.java

```diff
@@ -46,4 +52,18 @@ public void testInsertNullMap() throws SQLException {
 
         EasyMock.verify(statementMock);
     }
+    
+    @Test
+    public void testApproximateCountQueryAndBlankSchemaName() {
+        final SelectItem selectItem = SelectItem.getCountAllItem();
+        selectItem.setFunctionApproximationAllowed(true);
+        final Query query = new Query();
+        query.select(selectItem);
+        query.from(new MutableTable("tbl").setSchema(new MutableSchema("")));
+        assertEquals("SELECT APPROXIMATE COUNT(*) FROM tbl", query.toSql());
+        
```

Comment:
Again, not sure about the code style conventions, but lines 55 and 64 contain trailing spaces (which for some reason aren't visible in the browser view), can you remove these?